releasenotes/notes/qasm2-float-decimal-76b44281d9249f7a.yaml

```diff
@@ -0,0 +1,8 @@
+---
+fixes:
+  - |
+    Angles in the OpenQASM 2 exporter (:func:`.QuantumCircuit.qasm`) will now always include a
+    decimal point, for example in the case of ``1.e-5``.  This is required by the letter of the
+    floating-point-literal specification in OpenQASM 2.  Qiskit's OpenQASM 2 parser
+    (:func:`.qasm2.load` and :func:`~.qasm2.loads`) is more permissive by default, and will allow
+    ``1e-5`` without the decimal point unless in ``strict`` mode.
```

Comment:
I've actually never heard the phrase "required by the letter of" and wasn't sure how to interpret it in the PR description until I saw it with more context here 😄. Maybe it's worth changing to something less fun like "This is required by the OpenQASM 2 specification for floating point literals."

Reply:
The "letter of the law" is a common phrase in British English at least (as opposed to the "spirit of the law").  I changed it a little in 9828a5e14, if that's better?